### PR TITLE
initial commit of OUSD integration

### PR DESCRIPTION
**Pull request by sparrowDom**

### CODE REVIEW

File: contracts/adapters/ousd/OusdAssetAdapter.sol [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+pragma solidity 0.6.5;
+pragma experimental ABIEncoderV2;
+
+import { ERC20 } from "../../ERC20.sol";
+import { ProtocolAdapter } from "../ProtocolAdapter.sol";
+
+
+/**
+ * @title Asset adapter for Enzyme protocol.
```
Comment (sobolev-igor) [inline contracts/adapters/ousd/OusdAssetAdapter.sol:24]:
```suggestion
 * @title Asset adapter for OUSD token.
```

File: contracts/adapters/ousd/OusdAssetAdapter.sol [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * @dev Implementation of ProtocolAdapter interface.
+ * @author Domen Grabec
```
Comment (sobolev-igor) [inline contracts/adapters/ousd/OusdAssetAdapter.sol:26]: why didn't you leave an email?)

File: contracts/adapters/ousd/OusdAssetAdapter.sol [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ */
+contract EnzymeAssetAdapter is ProtocolAdapter {
```
Comment (sobolev-igor) [inline contracts/adapters/ousd/OusdAssetAdapter.sol:28]:
```suggestion
contract OusdAssetAdapter is ProtocolAdapter {
```

**Review by sobolev-igor — CHANGES_REQUESTED** [review]

**Comment by sobolev-igor** [comment]
All in all, it looks great and I'll be ready to merge and deploy the contracts after the minor fixes I've mentioned)

File: contracts/adapters/ousd/OusdAssetAdapter.sol (hunk 2 above)
Comment (sparrowDom) [reply contracts/adapters/ousd/OusdAssetAdapter.sol:26]: good point adding tnx

**Pull request closed without merging**